recipes-openxt/surfman/surfman.inc

```diff
@@ -1,4 +1,4 @@
 PV = "0+git${SRCPV}"
 
-SRCREV = "${AUTOREV}"
+SRCREV = "46bfde5850d92399d36edb24313f2e404d53d4dd"
```

Comment:
Doesn't need to be done now, but this recipe should be deleted.

Reply:
I can only think it was kept for "optional build" purposes.